src/harness/rwcRunner.ts

```diff
@@ -170,29 +170,29 @@ namespace RWC {
 
 
             it("has the expected emitted code", () => {
-                Harness.Baseline.runBaseline(`${baseName}.output.js`, () => {
-                    return Harness.Compiler.collateOutputs(compilerResult.files);
-                }, baselineOpts);
+                Harness.Baseline.runMultifileBaseline(baseName, "", () => {

```

Comment:
technically we should clone these tests so `runBaseline` doesn't break, right? If we intend to stop using `runBaseLine`, maybe we should just delete it.

Reply:
`runBaseline` is still used for tons of not-rwc-tests, so it's not like it is going away.